### DISCUSSION TITLE
Bug/Question about glow module

### DISCUSSION THREAD

**Discussion by NoltoxGit** [Zrips/CMI#8880] [Enhancement]
### Description of Issue

Helllo,
I've a bug/question about the glow module,

I use [CustomNameplates (Source code here)](https://github.com/Xiao-MoMi/Custom-Nameplates) for the unlimited player nameplates. And when a player choose a glow color, their show a nameplates of CMI, so all players see double nameplates.

## So one possible solution that we thought of with the developer of this plugin would be that when a player chooses a glow, it hides their nale (with a parameter in the configuration about this).

Screen of the bug:
![image](https://github.com/Zrips/CMI/assets/75265787/dd1dcd75-b0ae-4e5b-9b80-fd8740f589a6)

### Version Information

```shell
CMI: 9.7.3.3 Velocity CMIB MySQL
CMILib: 1.4.7.16
Server: Purpur(2176) 1.20.4-R0.1-SNAPSHOT-
CMI economy: Activée Vault: 1.7.3-CMI CMI Chat: Désactivée
Modules -> 31 enabled 33 disabled: spawnerProximity, cuffed, mirror, tablist, armorEffects, moneyCheque, elytraBoost, elytraLaunch, bossBarCompass, chatBubble, votifier, headDrop, spawnerCharge, vanish, homeInteractions, hpBossBar, anvilRenameColor, jail, spawners, nightSpeedup, dynamicSigns, playerChatTag, durabilityLoss, deathMessages, teleportWith, disabledEnchants, worldLimitsElytra, worldLimits, ranks, sitAnimation, silentChest, warmup, totemBehavior
```

### Relevant Plugins

CustomNameplates 2.3.3.9

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
CMI nameplate and CMI Glow only adjusts the value, the prefix/suffix/color, it doesn't have a hide or show nameplate value or feature.  I think this is currently not possible with CMI.

**Comment by NoltoxGit** [comment]
> CMI nameplate and CMI Glow only adjusts the value, the prefix/suffix/color, it doesn't have a hide or show nameplate value or feature. I think this is currently not possible with CMI.

Hmmm, do you think it's possible to hide nameplates with /cmi nameplates ?

**Comment by mrfloris** [comment]
Currently not a feature, up to zrips to add that if it's possible.

**Comment by NoltoxGit** [comment]
> Currently not a feature, up to zrips to add that if it's possible.

I think this could be a solution, as my glow goes through a custom interface, so I could make `/cmi nameplates "&f"` for example.
